<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>enterPoReceipt</class>
 <widget class="QWidget" name="enterPoReceipt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Order Receipts</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>12</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <layout class="QHBoxLayout">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <item>
            <widget class="OrderCluster" name="_order">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="topMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QCheckBox" name="_dropShip">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Drop Ship</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="XCheckBox" name="_singleLot">
           <property name="text">
            <string>Assign all Lot-controlled Items to single Lot #</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_print">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Print </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_post">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>P&amp;ost</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_save">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_orderitemLit">
            <property name="text">
             <string>Order Line Items:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="XTreeWidget" name="_orderitem"/>
            </item>
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="_printLabel">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Print &amp;Label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_enter">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&amp;Enter Receipt</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_all">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Receive &amp;All</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="_bcLit">
         <property name="text">
          <string>Receive Items using bar code:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XLineEdit" name="_bc"/>
       </item>
       <item>
        <widget class="QLabel" name="_bcQtyLit">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Qty.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XLineEdit" name="_bcQty">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XCheckBox" name="_autoPost">
         <property name="text">
          <string>Auto Post</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_bcFind">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>OrderCluster</class>
   <extends>QWidget</extends>
   <header>ordercluster.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_singleLot</tabstop>
  <tabstop>_orderitem</tabstop>
  <tabstop>_enter</tabstop>
  <tabstop>_all</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_post</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>enterPoReceipt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_orderitem</sender>
   <signal>itemSelected(int)</signal>
   <receiver>_enter</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_orderitem</sender>
   <signal>valid(bool)</signal>
   <receiver>_enter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_order</sender>
   <signal>valid(bool)</signal>
   <receiver>_print</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_order</sender>
   <signal>valid(bool)</signal>
   <receiver>_save</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_order</sender>
   <signal>valid(bool)</signal>
   <receiver>_all</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_order</sender>
   <signal>valid(bool)</signal>
   <receiver>_post</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_orderitem</sender>
   <signal>valid(bool)</signal>
   <receiver>_printLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_bc</sender>
   <signal>returnPressed()</signal>
   <receiver>_bcFind</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
